components/identity-core/org.wso2.carbon.identity.core/pom.xml

```diff
@@ -74,6 +74,14 @@
             <groupId>org.wso2.securevault</groupId>
             <artifactId>org.wso2.securevault</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.commons</groupId>
+            <artifactId>org.wso2.carbon.tenant.common</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.identity</groupId>
+            <artifactId>org.wso2.carbon.identity.user.store.configuration</artifactId>
+        </dependency>
```

Comment:
This introduces cyclic dependency, user.store.configuration is depending on identity.core, we can't put identity.core to depend on it.